app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -293,12 +353,9 @@ class ParkingViewModel(
               TurfMeasurement.distance(_circleCenter.value!!, parking.location.center)
             }
             .filter { parking ->
-              (_selectedProtection.value.isEmpty() ||
-                  _selectedProtection.value.contains(parking.protection)) &&
-                  (selectedRackTypes.value.isEmpty() ||
-                      _selectedRackTypes.value.contains(parking.rackType)) &&
-                  (_selectedCapacities.value.isEmpty() ||
-                      _selectedCapacities.value.contains(parking.capacity)) &&
+              _selectedProtection.value.contains(parking.protection) &&
```

Comment:
I just realized that this filter could be integrated with the third one and it could make the sorting faster if we have less elements in the list.

Reply:
Hi, I don't really understand what you mean by this. Could you maybe rephrase? Did you mean including this in the filter by distance with &&? I don't know how much we could benefit from it but I can do this, doesn't seem too hard. Is this also this that you meant when you said "unifying the filters"

Reply:
Yes, sorry if I was unclear, but yes, the idea was to extend the predicate we have for the first filler